From 7183b9ec8c6b3f536928a5341d6fd9ee37e05276 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Tue, 9 Oct 2018 11:37:56 +0800
Subject: [PATCH 1/1] remove this unused function.

major and minor are missed in the latest libs, so remove this unused function.

Upstream-Status: Inappropriate [TI specific]

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 gbm.c    | 26 --------------------------
 gbmint.h |  3 ---
 2 files changed, 29 deletions(-)

diff --git a/gbm.c b/gbm.c
index c046b1a..cd8f63f 100644
--- a/gbm.c
+++ b/gbm.c
@@ -102,32 +102,6 @@ gbm_device_destroy(struct gbm_device *gbm)
       gbm->destroy(gbm);
 }
 
-struct gbm_device *
-_gbm_mesa_get_device(int fd)
-{
-   struct gbm_device *gbm = NULL;
-   struct stat buf;
-   dev_t dev;
-   int i;
-
-   if (fd < 0 || fstat(fd, &buf) < 0 || !S_ISCHR(buf.st_mode)) {
-      errno = EINVAL;
-      return NULL;
-   }
-
-   for (i = 0; i < device_num; ++i) {
-      dev = devices[i]->stat.st_rdev;
-      if (major(dev) == major(buf.st_rdev) &&
-          minor(dev) == minor(buf.st_rdev)) {
-         gbm = devices[i];
-         gbm->refcount++;
-         break;
-      }
-   }
-
-   return gbm;
-}
-
 /** Create a gbm device for allocating buffers
  *
  * The file descriptor passed in is used by the backend to communicate with
diff --git a/gbmint.h b/gbmint.h
index 155eb12..90289d2 100644
--- a/gbmint.h
+++ b/gbmint.h
@@ -111,7 +111,4 @@ struct gbm_backend {
    struct gbm_device *(*create_device)(int fd);
 };
 
-struct gbm_device *
-_gbm_mesa_get_device(int fd);
-
 #endif
-- 
2.7.4

